### PR TITLE
chore(shared): Deprecate useOrganizations

### PR DESCRIPTION
**Pull request by panteliselef**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [x] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.

The `useOrganizations` only contains logic that can be directly exported from `useClerk()` and `useOrganization` and `useOrganizationList` can be used as replacement with more functionality and better DX

### CODE REVIEW

**Review by dimkl — APPROVED** [review]

**Review by anagstef — APPROVED** [review]

File: .changeset/nervous-poets-remember.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/shared': minor
+---
+
+Deprecate the `useOrganization` react hook. The hook can be replaced from useClerk, useOrganization, useOrganizationList
```
Comment (nikosdouvlis) [inline .changeset/nervous-poets-remember.md:5]: Should this be `useOrganizations`?

**Review by nikosdouvlis — APPROVED** [review]
Might have a minor typo, other than that this LGTM

File: .changeset/nervous-poets-remember.md (hunk 1 above)
Comment (panteliselef) [reply .changeset/nervous-poets-remember.md]: Thanks for catching this one

**Comment by clerk-cookie** [comment]
This PR has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.

**Pull request merged**